packages/kbn-es-query/src/kuery/functions/and.js

```diff
@@ -25,13 +25,13 @@ export function buildNodeParams(children) {
   };
 }
 
-export function toElasticsearchQuery(node, indexPattern) {
+export function toElasticsearchQuery(node, indexPattern, config = null) {
```

Comment:
I prefer to set defaults for those reading the code. It's better to be explicit as to what is optional and what is not.

Reply:
We could add a doc comment or a Typescript definition to make it more explicit, or we could default `config` to an empty object. I just like to avoid using `null` as much as possible since it complicates any code that has to handle it.